app/client/ui2018/checkbox.ts

```diff
@@ -106,7 +106,6 @@ export const cssCheckboxCircle = styled(cssCheckboxSquare, `
 `);
 
 export const cssLabelText = styled('span', `
-  margin-left: 8px;
```

Comment:
Same comment as below. Can you just copy this style locally or extend it?

Reply:
I revert this file to origin/main state.
And unify style in document settings between `template mode` and `Chronometer` modals.